package.json

```diff
@@ -33,7 +33,9 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
-    "moment": "^2.24.0"
+    "lodash": "^4.17.20",
+    "moment": "^2.24.0",
```

Comment:
if we don't need very explicit versions here, maybe we can make this super generalized (you don't have to do this now), but maybe we can just do moment >2 and lodash >4